src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/TimingsHack.java

```diff
@@ -256,9 +257,10 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
 				view.getRenderers().forEach(view::removeRenderer);
 				view.addRenderer(this.tickVisualize);
 
-				ItemStack viewMap = new ItemStack(Material.MAP, 1, view.getId());
+				ItemStack viewMap = new ItemStack(Material.MAP, 1);
```

Comment:
I need to review this whole structure here. Might be a better way now, with API rewrites.